automated/linux/armnn-benchmarks/armnn-mlperf.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    l) LINK_SNAPSHOT="${OPTARG}" ;;
+    m) MLPERF="${OPTARG}" ;;
+    a) DATASET0="${OPTARG}" ;;
+    b) DATASET1="${OPTARG}" ;;
+    c) DATASET2="${OPTARG}" ;;
+    d) DATASET3="${OPTARG}" ;;
+    e) DATASET4="${OPTARG}" ;;
+    f) DATASET5="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
+
+pkgs="ntp git wget curl libz-dev gcc g++ autoconf autogen libtool python3 python3-pip libjpeg-dev libtinfo5 libncurses5-dev libncursesw5-dev libboost-all-dev"
+dhclient
+install_deps "${pkgs}"
+
+wget $SNAPSHOT
+tar xf armnn.tar.xz
+cd home/buildslave/workspace/armnn-ci-build
+export BASEDIR=`pwd`
+cd $BASEDIR/armnn/build
+ln -s $BASEDIR/protobuf-host/lib/libprotobuf.so.15.0.0 ./libprotobuf.so.15
+export LD_LIBRARY_PATH=`pwd`
+chmod a+x UnitTests
```

Comment:
Are UnitTests used anywhere?

Reply:
I've added a line to execute the UnitTests.